### PR TITLE
Adjusted Quick Start Intro

### PR DESCRIPTION
**Pull request by shawazi**
# Pull Request

## Related issue
Fixes #<issue_number>

## What does this PR do?

I thought it might improve readability to:
 
- Convert intro paragraph
- To bullet points

## PR checklist
Please check if your PR fulfills the following requirements:
- [X] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [ ] Have you read the contributing guidelines?
- [X] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Review by guimachiavelli — CHANGES_REQUESTED** [review]
Hello @shawazi! Thanks so much for your interest in contributing to our repository!

The @meilisearch/docs-team discussed this PR and would like to request a few changes. This is due more to broader editorial decisions than the quality of your suggestions.

Additionally, we just released a major new version of the docs. Could you rebase your PR with the `main` branch to avoid merge conflicts?

Thanks again!

File: learn/getting_started/quick_start.md [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
+- Some way to interact with Meilisearch afterwards (for example, [cURL](https://curl.se) or one of our [SDKs](/learn/what_is_meilisearch/sdks.md))
 
 Let's get started!
 
 ## Setup and installation
 
-We'll start with downloading and installing Meilisearch. You have the option to install Meilisearch locally or deploy it over a cloud service.
+We'll start with downloading and installing Meilisearch. 
+
+You have the option to install Meilisearch locally or deploy it over a cloud service.
```
[the remark this answers is not in the document]
Comment (shawazi) [reply learn/getting_started/quick_start.md]: My apologies for taking up your time with this.

I just made the commit rolling back each change I made.

**Comment by shawazi** [comment]
Thank you @guimachiavelli, I misinterpreted something and I will create a new pr with these changes: 

```
You will need:
- A [command line](https://www.learnenough.com/command-line-tutorial#sec-running_a_terminal) for installation
- Some way to interact with Meilisearch afterwards (for example, [cURL](https://curl.se) or one of our [SDKs](/learn/what_is_meilisearch/sdks.md))
```

**Comment by shawazi** [comment]
I have made some mistakes in this pr, so I'm going to abandon it and open a new pr with only the changes that were approved. Thank you

**Pull request closed without merging**